### ISSUE TITLE
dataset preparation emotions

### ISSUE THREAD

**Issue by csokosgeza** [canopyai/Orpheus-TTS-0.1#217]
When we prepare the dataset, 3 columns remain in the dataset: columns_to_keep = ["input_ids", "labels", "attention_mask"]
What is the "labels" column for?
After the dataset preparation script, the input_ids and labels are the same.
Could this column be used to describe the emotional nature of the sound?
